packages/create-cloudflare/src/helpers/packages.ts

```diff
@@ -11,7 +12,20 @@ import whichPmRuns from "which-pm-runs";
   - dlx: executing packages that are not installed locally (ex. `pnpm dlx create-solid`)
 */
 export const detectPackageManager = () => {
-	const { name, version } = whichPmRuns() ?? { name: "npm", version: "0.0.0" };
+	let { name, version } = whichPmRuns() ?? { name: "npm", version: "0.0.0" };
+
+	if (process.env.TEST_PM) {
```

Comment:
just to understand things better, I don't see you setting this anywhere, so I assume that it isn't currently used right?

Reply:
In package.json:
`"test:e2e:npm": "npm run build && TEST_PM=npm vitest run --config ./vitest-e2e.config.ts",`

Reply:
ah ok, sorry, somehow I missed that 🤦